packages/web/components/control/menu-dropdown-icon/menu-dropdown-icon-item.tsx

```diff
@@ -0,0 +1,51 @@
+import { FunctionComponent } from "react";
+import Image from "next/image";
+import classNames from "classnames";
+import React from "react";
+import { useTranslation } from "react-multi-lang";
+
+interface Props {
+  value: string;
+  display: string;
+  currentValue: string;
+  image: string;
+  index: number;
+  optionLength: number;
+  onSelect: ({ value }: { value: string }) => void;
+}
+
+export type MenuDropdownIconItem = {
+  value: string;
+  display: string;
+  image: string;
+};
+export const MenuDropdownIconItem: FunctionComponent<Props> = ({
+  value,
+  display,
+  image,
+  onSelect,
+  index,
+  currentValue,
+  optionLength,
+}: Props) => {
+  const t = useTranslation();
+  return (
+    <button
+      className={classNames(
+        "px-4 py-2 cursor-pointer hover:bg-osmoverse-700 flex items-center ",
+        {
+          "text-rust-200": value === currentValue,
```

Comment:
Don't believe the text rust applies to this type of dropdown.

See:
https://www.figma.com/file/kYSFzlWoiMFrqkF46dtZQV/Osmosis-Design-Guidelines-2022?node-id=579%3A4778